lib/api/src/modules/stories.ts

```diff
@@ -145,11 +145,8 @@ export const init: ModuleFn = ({
     getCurrentParameter: (parameterName) => {
       const { storyId, refId } = store.getState();
       const parameters = api.getParameters({ storyId, refId }, parameterName);
-
-      if (parameters) {
-        return parameters;
-      }
-      return undefined;
+      // FIXME I don't know why this is needed
+      return parameters || undefined;
```

Comment:
Is this some typescript issue or what's going on?

Reply:
Improved the comment